src/components/Profile.vue

```diff
@@ -1,10 +1,8 @@
 <template id="profile">
 
     <div class="profile-bar">
-        <div>Managing <router-link v-bind:to="'/user'"><strong>{{ selectedTrader.name }}</strong></router-link></div>
-        <transition name="fade" mode="out-in">
-            <div :key="voucherCount" v-if="vouchersAdded"><router-link v-bind:to="'/payment'"><strong>{{ voucherCount }}</strong> voucher<span v-if="voucherCount > 1">s</span> waiting</router-link></div>
-        </transition>
+        <div><router-link v-bind:to="'/account'"><strong>{{ selectedTrader.name }}</strong></router-link> <router-link v-bind:to="'/user'">(Change trader)</router-link></div>
```

Comment:
I don't think it made sense to have the trader name serving as the link to the change trader page, which is why I added the Change Trader link.

You're right, we don't necessarily need to have the trader name linking to the account page, but since there's no other place I can think of that it should link to, I don't think there's any harm in having another link to the account page.

I can remove the link though if you like and just leave it as the trader name. Let me know what you think!